docs/index.html

```diff
@@ -51,17 +51,6 @@ <h3 class="sidebar-header">COMPONENTS</h3>
             <li class="docs-navbar-item" target="alert">
```

Comment:
Here **target**  can be only used for anchor tags and doesn't have any purpose with **list** tags

Reply:
please look at it now @MaruthiPothuganti